### PR TITLE
Fixes for makeAuthRequest(...) and redirectToSignIn(...)

### PR DESCRIPTION
**Pull request by zone117x**
Backwards compatibility fixes:
* Fixes https://github.com/blockstack/blockstack.js/issues/618
* Fixes `makeAuthRequest(...)` missing in v19

### CODE REVIEW

**Review by yknl — APPROVED** [review]
Thanks for the fix! Looks good to go

File: src/auth/authApp.js [hunk 1]
```diff
@@ -166,12 +134,12 @@ export function signUserOut(redirectURL: ?string = null) { // eslint-disable-lin
  * by special authenticators.
  * @return {String} the authentication request
  */
-export function makeAuthRequest(transitPrivateKey: string,
-                                redirectURI: string,
-                                manifestURI: string,
-                                scopes: Array<string>,
+export function makeAuthRequest(transitPrivateKey: ?string,
+                                redirectURI: string = `${window.location.origin}/`,
```
Comment (kantai) [inline src/auth/authApp.js:138]: These defaults make this function un-callable from non-browser contexts. It makes sense that something like `redirectToSignIn` wouldn't work outside of a browser context, but `makeAuthRequest` seems like it should.

Comment (kantai) [reply src/auth/authApp.js:138]: Actually, nevermind -- it seems like `makeAuthRequest` is deprecated anyways, and non-browser apps would use the UserSession API anyways.

**Review by kantai — APPROVED** [review]
This looks good to me @zone117x

**Review by hstove — APPROVED** [review]
Dang.. good catch and thanks for fixing this.

**Pull request merged**